rust/feature-flags/src/server.rs

```diff
@@ -24,11 +24,20 @@ where
         }
     };
 
-    let read_postgres_client =
-        match get_pool(&config.read_database_url, config.max_pg_connections).await {
+    let postgres_reader = match get_pool(&config.read_database_url, config.max_pg_connections).await
+    {
+        Ok(client) => Arc::new(client),
+        Err(e) => {
+            tracing::error!("Failed to create read Postgres client: {}", e);
+            return;
+        }
+    };
+
+    let postgres_writer =
+        match get_pool(&config.write_database_url, config.max_pg_connections).await {
```

Comment:
(note for Dylan): add a TODO here to change the write_database_url to point to the writer (and to have the read_database_url point to the replica), since having these two clients is pointless if they're connecting to the same DB host.